baseplate/core.py

```diff
@@ -581,6 +584,10 @@ def service(self):
         """:py:class:`~baseplate.core.Service` object for the current context."""
         return Service(self.authentication_token)
 
+    @cached_property
```

Comment:
is there any value to this being a `cached_property`? the underlying `_t_request` already is and then from there it's just an attribute access.